packages/core/src/state-management/state-manager.ts

```diff
@@ -197,7 +197,8 @@ export class StateManager {
           }
         }),
         catchError((error) => {
-          this.logger.err(`Error while anchoring stream ${state$.id.toString()}:\n${error}`)
+          this.logger.err(`Error while anchoring stream ${state$.id.toString()}:${error}`)
```

Comment:
It seems like logging the error like this does not include the stack trace in the output, while logging the full error object does.  Ideally this could be done in a single log statement so that there's no chance of the two log messages getting separated in the log file with other messages coming in between.  Perhaps @ukstv knows how best to do that?  I found the `Error.stack` object exists, which seems like exactly what we would want, but according to [the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack) it's a non-standard feature that can't always be relied on?  I'm not sure how big a deal that actually is or if we can safely use it anyway.